.github/CODEOWNERS

```diff
@@ -329,6 +329,8 @@ app/models/transaction_notification.rb @department-of-veterans-affairs/vfs-authe
 app/models/triage_team.rb @department-of-veterans-affairs/vfs-mhv-secure-messaging @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 app/models/all_triage_teams.rb @department-of-veterans-affairs/vfs-mhv-secure-messaging @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 app/models/user_acceptable_verified_credential.rb @department-of-veterans-affairs/octo-identity
+app/models/user_action.rb @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
```

Comment:
These all aren't quite in alphabetical order. 

Also, please only put `@department-of-veterans-affairs/octo-identity` on each line, not links to the other teams